src/components/AddNewBook.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books';
+import './addNewBook.css';
+
+const AddNewBook = () => {
```

Comment:
**Changes Required**
- [ ] The component should contain: - A local state with `title`, `author`. Every time a user changes the input value of either `title` or `author`, react should re-render in response to the state update. 
You can then pass the value of title and author to the function handling onSubmit to dispatch the action to the store.